### PR TITLE
Milestone 2 : React Calculater

### PR DESCRIPTION
**Pull request by RailonA**
Implement 4 components: 

- App
- Display
- Button Panel
- Button

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable func-style */
```
Comment (whiz25) [inline src/components/app.js]: Fix this linter error. Disabling the linter rules changes the linter config which is used by all Microverse students.

File: src/components/app.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import ButtonPanel from "./buttonPanel";
+import Display from "./display";
+import React from "react";
+
+function App () {
+
+    return (
+        <div>
```
Comment (whiz25) [inline src/components/app.js]: Use React.Fragment shorthand syntax
```
<>...</>
```

File: src/components/display.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable func-style */
```
Comment (whiz25) [inline src/components/display.js]: Fix these linter errors

File: src/components/button.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable func-style */
```
Comment (whiz25) [inline src/components/button.js]: Fix the linter error instead of disabling the rule.

File: src/components/buttonPanel.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
+/* eslint-disable max-lines-per-function */
```
Comment (whiz25) [inline src/components/buttonPanel.js:1]: Fix the linter errors

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work in this milestone. I have come a long way. Well done.
### Changes 🔕 🔕 🔕 
- add `.vscode` folder to `.gitignore` file. This folder contains sensitive information about your project. It may not be a good idea to commit it.
- check comments for suggestions.


### Martin Afani
#### Slack: whiz25

**Review by bellom — APPROVED** [review]
## Hi @RailonA  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best**

**Pull request merged**